src/simcity.rs

```diff
@@ -0,0 +1,220 @@
+        "Inserting Sublimated Messages",
+        "Integrating Curves",
+        "Integrating Illumination Form Factors",
+        "Integrating Population Graphs",
+        "Iterating Cellular Automata",
+        "Lecturing Errant Subsystems",
+        "Mixing Genetic Pool",
+        "Modeling Object Components",
+        "Mopping Occupant Leaks",
+        "Normalizing Power",
+        "Obfuscating Quigley Matrix",
+        "Overconstraining Dirty Industry Calculations",
+        "Partitioning City Grid Singularities",
+        "Perturbing Matrices",
+        "Pixalating Nude Patch",
+        "Polishing Water Highlights",
+        "Populating Lot Templates",
+        "Preparing Sprites for Random Walks",
+        "Prioritizing Landmarks",
+        "Projecting Law Enforcement Pastry Intake",
+        "Realigning Alternate Time Frames",
+        "Reconfiguring User Mental Processes",
+        "Relaxing Splines",
+        "Removing Road Network Speed Bumps",
+        "Removing Texture Gradients",
+        "Removing Vehicle Avoidance Behavior",
+        "Resolving GUID Conflict",
+        "Reticulating Splines",
+        "Retracting Phong Shader",
+        "Retrieving from Back Store",
+        "Reverse Engineering Image Consultant",
+        "Routing Neural Network Infanstructure",
+        "Scattering Rhino Food Sources",
+        "Scrubbing Terrain",
+        "Searching for Llamas",
+        "Seeding Architecture Simulation Parameters",
+        "Sequencing Particles",
+        "Setting Advisor Moods",
+        "Setting Inner Deity Indicators",
+        "Setting Universal Physical Constants",
+        "Sonically Enhancing Occupant-Free Timber",
+        "Speculating Stock Market Indices",
+        "Splatting Transforms",
+        "Stratifying Ground Layers",
+        "Sub-Sampling Water Data",
+        "Synthesizing Gravity",
+        "Synthesizing Wavelets",
+        "Time-Compressing Simulator Clock",
+        "Unable to Reveal Current Activity",
+        "Weathering Buildings",
+        "Zeroing Crime Network"
+    ];
+
+    let mut rng = thread_rng();
+
+    for _ in 0..500 {
+        let sleep_length = rng.gen_range(200, 10000);
+
+        // A number between 1 and 66 (inclusive)
+        let mut spinner_id: u8 = ((rng.next_u32() % 66) as u8) + 1;
+        let spinner_enum = match spinner_id {
+            1 => Spinners::Dots,
+            2 => Spinners::Dots2,
+            3 => Spinners::Dots3,
+            4 => Spinners::Dots4,
+            5 => Spinners::Dots5,
+            6 => Spinners::Dots6,
+            7 => Spinners::Dots7,
+            8 => Spinners::Dots8,
+            9 => Spinners::Dots9,
+            10 => Spinners::Dots10,
+            11 => Spinners::Dots11,
+            12 => Spinners::Dots12,
+            13 => Spinners::Line,
+            14 => Spinners::Line2,
+            15 => Spinners::Pipe,
+            16 => Spinners::SimpleDots,
+            //17 => Spinners::SimpleDotsScrolling,
```

Comment:
Why did you opt to not show some spinners? I think in that case you should just remove these commented lines and comment above that you don't want to use all of them as they don't seem appropriate to you.

Reply:
Some spinners did not work correctly here, so I commented them out instead of removing them, for completeness.

Updated the branch so that the comments are now removed.